phase4-lib/src/main/java/com/helger/phase4/crypto/AS4CryptParams.java

```diff
@@ -49,7 +52,6 @@
 public class AS4CryptParams implements Serializable, ICloneable <AS4CryptParams>
 {
   public static final ECryptoKeyIdentifierType DEFAULT_KEY_IDENTIFIER_TYPE = ECryptoKeyIdentifierType.BST_DIRECT_REFERENCE;
-  public static final String DEFAULT_KEY_ENC_ALGORITHM = WSS4JConstants.KEYTRANSPORT_RSAOAEP_XENC11;
```

Comment:
Can you please keep this constant and switch it to type `ECryptoKeyEncryptionAlgorithm` ?